stan/math/prim/meta/compiler_attributes.hpp

```diff
@@ -2,13 +2,48 @@
 #define STAN_MATH_PRIM_META_COMPILER_ATTRIBUTES_HPP
 
 #ifdef __GNUC__
+#ifndef likely
```

Comment:
Is it that old gccs didn't have likely and new ones do?

Reply:
Another package could have it